src/lint/lint.ts

```diff
@@ -320,8 +320,9 @@ async function fix(
       await applyEdits(edits, makeParseLoader(analyzer, analysis));
 
   for (const [newPath, newContents] of editedFiles) {
+    const documentPath = resolvePath(newPath);
```

Comment:
I forgot! We added a method on FsUrlLoader to deal with this: `FsUrlLoader#getFilePath()`

That might mean that we'd have to pass down an FsUrlLoader into this function (I think this collection of functions has enough shared state that it should be a class, but that's a different issue)

Reply:
oh nice! that makes the whole thing much simpler, we don't need to pass the config around but rather the urlLoader 👌 Fixed!